pkgs/development/libraries/sfml/sfml_git.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, mesa, libX11, freetype, libjpeg, openal, libsndfile
+, glew, libXrandr, libXrender
+}:
+stdenv.mkDerivation rec {
+  name = "sfml-git-20110428";
```

Comment:
So: remove sfml_git, and update the MARS package (which depends on sfml_git)?

Sounds good, I updated the MARS package to a higher version too, had to apply some patches to it (nothing big). Should I open a new pull request for the MARS update?